lib/auth/token/jwt.dart

```diff
@@ -17,11 +21,30 @@ class Jwt {
 
   String get userId => _payload.subject;
 
-  Jwt(String token) : _tokenParts = token.split(' ').last.split('.');
+  Jwt(this.token) : _tokenParts = token.split(' ').last.split('.');
+
+  Future<bool> validate(String projectId, Map<String, String> certificates,
+      {bool enforceEmailVerification = false,
+      bool checkRevoked = false}) async {
+    // Google can validate the token for us and we can compare the results to revoke it.
+    // Otherwise, do local validation so we don't use network bandwidth and deal with latency.
+    if (checkRevoked) {
+      var user = await FirebaseAuth.instance.getUser(uid: userId);
```

Comment:
You can't guarantee that the instance is initialised here.

You can create an instance of FirebaseAuth with a volatile TokenStore just to perform that request.

Reply:
I'll get on that.

Reply:
Can I just `assert` that firebase auth's instance is not initialized? Because as of right now, FirebaseAuth's initialize needs the firebase api key, and I don't want to have to pass that to `validate`. It would be redundant to need to pass the firebase key again here, or even passing the firebase auth instance. 
Or maybe I give the option to pass the firebase auth instance, and assert that either that instance is not null or firebase auth's global instance is not null, ONLY if you are checking if the token is revoked.